integration-cli/docker_cli_kill_test.go

```diff
@@ -33,6 +33,28 @@ func (s *DockerSuite) TestKillContainer(c *check.C) {
 	}
 }
 
+func (s *DockerSuite) TestKillofStoppedContainer(c *check.C) {
+	runCmd := exec.Command(dockerBinary, "run", "-d", "busybox", "top")
+	out, _, err := runCommandWithOutput(runCmd)
+	if err != nil {
```

Comment:
Since we need a couple of changes anyway, would be great to use `c.Assert(err, check.IsNil)` and `c.Assert(err, check.Not(check.IsNil), check.Commentf("this should have failed")` for these error checks.